### PR TITLE
Add staff solutions for pset6

### PR DESCRIPTION
**Pull request by brianyu28**
@dmalan, this works in my IDE, would be great if someone could test just to be sure.

Fairly certain that these solution files can only be executed with Python 3.4.3 (the version installed in the IDE), since the same version of Python used to compile the bytecode needs to execute it.

### CODE REVIEW

**Comment by dmalan** [comment]
Will test! Mind integrating this change? https://github.com/cs50/solutions/commit/5a9364f56b3c371a7b2d3803821cb6ab4eac59d6

**Comment by brianyu28** [comment]
Integrated!

**Comment by dmalan** [comment]
Can't delete `server`, still needed for CS50x!

File: ide50/files/home/cs50/pset6/tweets [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python3 tweets.pyc "$@"c
```
Comment (dmalan) [inline ide50/files/home/cs50/pset6/tweets]: What's the trailing `c` for?

**Comment by dmalan** [comment]
Re 3.4.3, that sounds right. Though any chance there's a way to make this approach a bit more resilient? E.g., is there some way to specify "please make this work on a few more versions"? Guessing not but just in case!

File: ide50/files/home/cs50/pset6/tweets (hunk 1 above)
Comment (brianyu28) [inline ide50/files/home/cs50/pset6/tweets]: Oops, don't know how that got there.

**Comment by brianyu28** [comment]
Added `server` back and fixed the typo.

**Pull request merged**